### ISSUE TITLE
ReleaseClearingStateSummary in project/detail

### ISSUE THREAD

**Issue by heydenreich** [sw360/sw360portal#127]
1.  Release Clearing State Summary in project-details view misses the 5th field: number of approved releases.
2. If setting the clearing state of a release to "new" manually, then the release clearing state summary of the project is updated (in project view and project detail), but in the wrong way: instead of increasing the number of new releases, the number of releases "under clearing by project team" is increased.

Reason: the release holds also a map " ClearingTeamToFossologyStatus", where for each team the fossology status is listed. This map isn't updated when the clearing state is set manually.

Solution: the clearing state of a release should not be set manually, but should be automatically set.  (Already done for "report available" and "approved". Todo: for "under clearing"). 
The map  " ClearingTeamToFossologyStatus" can probably vanish if clearingRequests are introduced, see #128.

**Comment by heydenreich** [comment]
1.  Release Clearing State Summary in project-details view misses the 5th field: number of approved releases.
2. If setting the clearing state of a release to "new" manually, then the release clearing state summary of the project is update, but in the wrong way: instead of increasing the number of new releases, the number of releases "under clearing by project team" is increased.

**Comment by heydenreich** [comment]
part 1. is solved in PR#143

**Comment by mcjaeger** [comment]
will provide a status diagram for this.

**Comment by mcjaeger** [comment]
[NEW_CLEARING] -> [SENT_TO_FOSSOLOGY](user sends to fossology)

[SENT_TO_FOSSOLOGY] -> [UNDER_CLEARING](fossology status update queried by user)


[UNDER_CLEARING] -> [REPORT_AVAILABLE](user has created / uploaded attachment with type clearing report)

[REPORT_AVAILABLE] -> [APPROVED](attachment of type clearing report is approved using the controls in edit mode)

[APPROVED] -> [REPORT_AVAILABLE](approval for attachment of type clearing report is rejected
)
[APPROVED] -> [NEW_CLEARING](attachment of type clearing report is deleted)

[REPORT_AVAILABLE] -> [NEW_CLEARING](attachment of type clearing report is deleted)

**Comment by alex-evo** [comment]
@mcjaeger, is there a general rule for dealing with multiple clearing reports?

**Comment by mcjaeger** [comment]
Hello, seems like testing reveals that the issue is a bit more complicated. Let's keep it manually and then move it to 1.4 to have 1.3 out soon.

**Comment by mcjaeger** [comment]
@alex-evo no general rule. Actually, there could be multiple clearing report, if the users are not happy with one created at first time, or that it available. The issue is that multiple clearing report could be displayed better in listings (currently you just have the download icon circles per clearing report). Maybe there could be more about this download like when the user hovers over it, there is a line displayed who has uploaded the clearing report and when.

**Comment by alex-evo** [comment]
depends on #157

**Comment by mcjaeger** [comment]
from common discussion: note that the UI is not updated with reaching the status "send to fossology", so in the java pop up it is sent but in the main table view, old status is still visible.

**Comment by mcjaeger** [comment]
should be retested: observed was that in FOSSology setting back to open, in sw360 stat change is not reflected.

**Comment by alex-evo** [comment]
Such state transition was not in the requirements for this issue. See https://github.com/sw360/sw360portal/issues/127#issuecomment-215021527

**Comment by mcjaeger** [comment]
is not there anymore according to common discussion today
